sdk/java-sdk-spring/tempo-query.yaml

```diff
@@ -0,0 +1 @@
+backend: "tempo:3200"
```

Comment:
Are these only for the spring sdk integration tests, if not, should they be in more places?

Reply:
So far these are only for the Spring SDK. At the moment, that's the only one I have integration tests for.

Reply:
Ok 👍